src/corelibs/U2Gui/src/util/CreateAnnotationFullWidget.cpp

```diff
@@ -153,19 +154,38 @@ void CreateAnnotationFullWidget::setAnnotationName(const QString &name) {
 }
 
 void CreateAnnotationFullWidget::setLocation(const U2Location &location) {
+    // Example: (200..len,1..100)=(200..100), lambda returns true
+    auto isSimpleSplitLocation = [](const U2Location &location, qint64 seqLen) {
```

Comment:
You've captured variables by value (cuz this is the default behavior), but you have `const U2Location &location` - const reference, which make no sense in this case. It's better to capture variables by reference.